src/System.Linq.Queryable/tests/System.Linq.Queryable.Tests.builds

```diff
@@ -5,6 +5,11 @@
     <Project Include="System.Linq.Queryable.Tests.csproj" />
     <Project Include="System.Linq.Queryable.Tests.csproj">
       <OSGroup>Windows_NT</OSGroup>
+      <TestTFMs>net463</TestTFMs> 
+    </Project>
+    <Project Include="System.Linq.Queryable.Tests.csproj">
+      <OSGroup>Windows_NT</OSGroup>
+      <TargetGroup>netstandard1.3</TargetGroup>
```

Comment:
Copied from the `Microsoft.CSharp` example. What's the role of having `netstandard1.3` leftover here? I'm not entirely clear about the role of this file so am going at it in a copy/paste/hope manner.